extensions/amp-facebook/0.1/amp-facebook.js

```diff
@@ -41,11 +41,10 @@ class AmpFacebook extends AMP.BaseElement {
     const iframe = getIframe(this.element.ownerDocument.defaultView,
         this.element, 'facebook');
     this.applyFillContent(iframe);
+    const amp = this.element;
     // Triggered by context.updateDimensions() inside the iframe.
     listenFor(iframe, 'embed-size', data => {
-      iframe.height = data.height;
-      iframe.width = data.width;
-      const amp = iframe.parentElement;
+      // TODO REVIEW
```

Comment:
We're [directly setting](https://github.com/ampproject/amphtml/pull/4674/files#diff-9e9b3893a5559f26e8a5b06f9f959613R48) the `height` attribute before calling `#changeHeight`.